ide/coqide/coqide_ui.ml

```diff
@@ -98,9 +99,11 @@ let init () =
 \n    <menuitem action='Forward' />\
 \n    <menuitem action='Backward' />\
 \n    <menuitem action='Run to cursor' />\
+\n    <menuitem action='Reset' />\
```

Comment:
Did you intend to move Reset?  If so, why?

Reply:
Yes I did. One goal of a good GUI (as opposed to a CLI) is to make the user able to quickly understand and master many commands or features. When I saw “Reset” at this place, despite the icons and the corresponding hotkeys, it took me some time to make sure that it was really just the opposite of “Run to end,” because the labels really differ and there were other items in between (and even more: “Reset” was at the end of the “Navigation” items, so it could stand for something completely different). Moving it near “Run to end” solves this problem.

Another reason to this move is how I consider what you should do when discovering CoqIDE: you will start by moving forward or run to cursor (if your teacher tells you to skip what’s at the top), maybe you will go backward to focus on something (how it is before, how it is after, let’s see it again and again), and then you should reset everything (provided that you are not asked to skip what is at the top, but in the course I followed, there was rarely something to skip) to make sure you have understood everything well. Other commands will find their meaning later (in my course, we had no reason to use “Fully check” and “Interrupt”), so should be the last in the menu.

Putting “Reset” before “Run to end” might seem inconsistent with “Forward” being before “Backward,” but I think it’s better like that because “Run to end” might have something to do with “Fully check” and “Interrupt,” whereas “Reset” does not.